### PR TITLE
Freeze yarn to 1.21.1

### PR DESCRIPTION
**Pull request by rhymes**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

As mentioned in [Introducing Yarn 2 !](https://dev.to/arcanis/introducing-yarn-2-4eh1) on DEV by its maintainer and by @abenettt in #5740 Yarn 2 is out, and we don't support it, thus running the risk of breaking something in the future when Yarn 2 will be the default install.

As mentioned in the article:

> If you don't want to upgrade all of your projects, just run yarn policies set-version ^1 in the repositories that need to stay on Yarn 1, and commit the result. Yarn will always prefer the checked-in binaries over the global ones, making it the best way to ensure that everyone in your team shares the exact same release!

I did just that, we can revisit which yarn version and how to upgrade to 2 in the future, in the meantime we should enforce this

## Related Tickets & Documents

- #5740 
- [Enforcing Yarn’s version across your project](https://legacy.yarnpkg.com/en/docs/cli/policies#enforcing-yarns-version-across-your-project-)

### CODE REVIEW

**Comment by abenettt** [comment]
@rhymes I tried this too but when I executed the file to get the version like 
```bash
 .yarn/releases/yarn-1.21.1.js --version
``` 
I got told I'm on yarn `2.0.0-rc.27`

**Comment by rhymes** [comment]
@abenettt I think this change is to make sure `yarn install` picks up the correct yarn. 

What happens if you exit the terminal, go into the directory again and type `yarn -v` ?

**Comment by abenettt** [comment]
@rhymes I'm having the same result then.

**Comment by rhymes** [comment]
@abenettt mmm how did you install Yarn 2 then?

**Comment by abenettt** [comment]
@rhymes Aaaaah I ran
```bash
yarn policies set-version berry
```
in my home directory and apparently as the repo lies within a child directory it takes precedence. Now that I removed it, it works again.

File: .yarnrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (nickytonline) [inline .yarnrc]: I've never used yarnrc. Is it recommended when freezing a version to include the frozen version as part of your repo? It kind of looks like that based on the [yarnrc](https://legacy.yarnpkg.com/en/docs/yarnrc/) docs.

Comment (rhymes) [reply .yarnrc]: > I've never used yarnrc. Is it recommended when freezing a version to include the frozen version as part of your repo?

I think so. This is what yarn's maintainer suggested: 

> If you don't want to upgrade all of your projects, just run yarn policies set-version ^1 in the repositories that need to stay on Yarn 1, and commit the result. Yarn will always prefer the checked-in binaries over the global ones, making it the best way to ensure that everyone in your team shares the exact same release!

I just did that, ran the command, committed the result :)

see their article https://dev.to/arcanis/introducing-yarn-2-4eh1

**Pull request merged**